### PR TITLE
Preview images on the channel pages

### PR DESCRIPTION
**Pull request by elGatoMantocko**
This is a special pull request and could possibly be a no-merge, but I will try to outline it here as best I can.

- Our goal is to preview images on each individual channel page.
- We reduced the number of images fetched to 4 to reduce risk of breaking the quota.

This branch's biggest purpose was to really test the ability of the /channels/getUrls meteor method. While we accomplished our goal of identifying issues, I still think that using the google image fetcher may be our best course of action, given the time remaining to complete the project.

Problems encountered:

- quota limits
- ~~content security policy (browser-policy) for images outside of domain~~

### CODE REVIEW

**Comment by marti584** [comment]
:+1:

**Comment by elGatoMantocko** [comment]
So after adding the browser policy update, it looks like heroku is still having issues displaying the images. I am thinking I will add another browser policy to allow "*.*.*", but I don't think that will work.

File: server/methods/read/channels.js [hunk 1]
```diff
@@ -12,9 +12,8 @@ Meteor.methods({
     check(channel, Channel);
  
     images = [];
-    for (var i = 0; i < 16; i++) {
-      images = images.concat(Modules.server.googleImageSearchSync(channel.get('query')));
-    }
+    images = _.union(images, Modules.server.googleImageSearchSync(channel.get('query')));
+
     return images;
```
Comment (cjsauer) [inline server/methods/read/channels.js]: Can we just change this to `return Modules.server.googleImageSearchSync(channel.get('query')));` ?

**Comment by cjsauer** [comment]
I think it'd be wise to merge the changes of PR #23 into this before closing; if we still were planning on merging this PR that is.

File: server/methods/read/channels.js (hunk 1 above)
Comment (elGatoMantocko) [inline server/methods/read/channels.js]: Yeah I will change that

**Pull request merged**